apps/dashboard/src/components/welcome/ai-prompts/framework-prompts/nextjs-prompt.ts

```diff
@@ -0,0 +1,476 @@
+- **Authentication Errors**: Handle gracefully with fallbacks
+- **TypeScript Errors**: Enforce proper typing throughout
+- **Network Errors**: Implement retry logic and offline handling
+- **Component Loading Errors**: Provide fallback UI
+
+**Error Handling Implementation**:
+\`\`\`typescript
+// Example error handling
+if (!process.env.NEXT_PUBLIC_NOVU_APPLICATION_IDENTIFIER) {
+  console.warn('NEXT_PUBLIC_NOVU_APPLICATION_IDENTIFIER is not configured');
+  return <div>Notifications not configured</div>;
+}
+
+if (!subscriberId || subscriberId === 'USER_SUBSCRIBER_ID') {
+  console.warn('Subscriber ID not available');
+  return <div>Please log in to view notifications</div>;
+}
+\`\`\`
+
+### Step 10: Iterative Refinement Process
+**Objective**: Fine-tune the integration based on validation results
+
+**Refinement Areas**:
+- Adjust inline appearance properties
+- Optimize subscriber detection logic
+- Improve placement positioning
+- Preserve validated design tokens and placement
+- Enhance error handling
+- Optimize performance
+- Improve accessibility
+
+### Step 11: Final Output Requirements
+**Objective**: Deliver a complete, production-ready integration
+
+**Required Deliverables**:
+- Self-contained InboxIntegration.tsx component
+- Inline appearance prop with empty placeholders
+- Subscriber detection with fallback mechanism
+- Environment variable reference via .env.local
+- Optional icons placeholder for customization
+- TypeScript compliance with proper typing
+- Responsive design implementation
+- Dark mode support
+- Accessibility compliance
+- Error boundaries and loading states
+- Performance optimizations
+- Security best practices implementation
+
+---
+
+## 🔧 TROUBLESHOOTING GUIDE
+
+### Common Issues and Solutions:
+
+**1. Component Not Rendering**
+- Check environment variable configuration
+- Verify subscriber ID is properly set
+- Ensure proper import from @novu/nextjs
+- Check for TypeScript compilation errors
+
+**2. Styling Issues**
+- Verify appearance prop structure
+- Check for CSS conflicts with host application
+- Ensure responsive design breakpoints
+- Validate dark mode compatibility
+
+**3. Authentication Integration**
+- Verify auth hook implementation
+- Check subscriber ID extraction logic
+- Ensure proper error handling for auth states
+- Validate fallback mechanisms
+
+**4. Performance Issues**
+- Check bundle size impact
+- Verify lazy loading implementation
+- Monitor network requests
+- Optimize re-renders
+
+**5. Accessibility Problems**
+- Test keyboard navigation
+- Verify screen reader compatibility
+- Check contrast ratios
+- Validate focus management
+
+---
+
+## 🎯 CORE PRINCIPLE
+
+The Inbox component must feel native to the host application — fully functional, fully responsive, and accessible. It should integrate inline, respecting the host app's patterns, without requiring external files or default icon modifications.
+
+---
+
+## 📝 EXAMPLE IMPLEMENTATION
```

Comment:
Why do we need this duplication with the above implementation? It looks like a copy paste